frontend/src/Traveler/TravelerOverview.tsx

```diff
@@ -0,0 +1,28 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {TravelerModel} from "./TravelerModel";
+
+
+export default function TravelerOverview(){
+    const [travelers,setTravelers] = useState<TravelerModel[]>([]);
+
+
+    useEffect(() => {
+        fetchAllTravelers()
+    },[])
+
+    const fetchAllTravelers = () => {
+        axios.get("/api/traveler")
+            .then(response => response.data)
+            .then(setTravelers)
+    }
+    const travelerList = travelers.map(traveler => {
+        return <p key={traveler.name}>{traveler.name}</p>
```

Comment:
Beachte: wenn sie als key verwendet werden, dann müssen die Namen eindeutig sein.
Alternative: Id (z.B. eine uuid) als key im Traveler und als key nutzen

Reply:
ist erledigt, da ich noch nicht bei dem Post issue bin habe ich die uuid quasi händisch in die db eingetragen und den record und das mapping angepasst